source/Dialog.cpp

```diff
@@ -268,7 +265,7 @@ bool Dialog::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool i
 		if(boolFun)
 		{
 			DoCallback(okIsActive);
-			GetUI()->Pop(this);
+			GetUI()->PopThrough(this);
```

Comment:
I'm not sure, why change this? (Mb add a comment for the first or smth?)

Reply:
`Pop()` just removes the dialog, which now has a `Panel` child. `PopThrough` will remove the dialog, and all of its children. 

Without this, it would remove the dialog, but leave the text on the screen.